modules/ols-attaching-a-resource-object-to-your-query.adoc

```diff
@@ -0,0 +1,29 @@
+// This module is used in the following assemblies:
+// ols-using-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ols-attaching-a-resource-object-to-your-query_{context}"]
+= Attaching a resource object to your query
+
+This procedure explains how to attach a resource object to provided additional context for your chat query. 
+
+.Procedure
+
+. Navigate to a supported resource in the {ocp-product-title} web console. For example, click *Workloads* -> *Pods* and then click the name of a pod.
```

Comment:
We may want to list the supported resources. Would need to double check with Andy Pickering.

Reply:
+1

Reply:
@kyoto Erik and Joao suggested I check in with you to confirm the supported resources that can be used. Just wanted to check in with you to see what should be listed.

Reply:
The supported resources are CronJob, DaemonSet, Deployment, Job, Pod, ReplicaSet and StatefulSet (all under "Workloads") and also Alert (under "Observe").

Reply:
done